recipes/cglm/all/conanfile.py

```diff
@@ -64,25 +67,19 @@ def build(self):
     def package(self):
         copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         if self.options.header_only:
-            copy(self, "*", src=os.path.join(self.source_folder, "include"), dst=os.path.join(self.package_folder, "include"))
+            copy(self, "*",
+                 src=os.path.join(self.source_folder, "include"),
+                 dst=os.path.join(self.package_folder, "include"))
         else:
             cmake = CMake(self)
             cmake.install()
-
             rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
             rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
-        self.cpp_info.set_property("cmake_file_name", "cglm")
-        self.cpp_info.set_property("cmake_target_name", "cglm::cglm")
```

Comment:
Please don't remove this, this lib exports config files. It's a nice information for maintainers that this recipe is aware of these names, even though they incidentally match default names of CMakeDeps.